File: joey_takes_money/joey.py
def main():
    cn = int(input())
    for _ in range(cn):
        n = int(input())
        sum = 1
        nums = list(map(int, input().split()))
        for num in nums:
            sum *= num
        sum += n - 1
        print(2022 * sum)
    
    
if __name__ == "__main__":
    main()

File: joey_takes_money/joey.cpp
#include <iostream>

using namespace std;

int main()
{
    int cn;
    cin >> cn;
    for (int i = 0; i < cn; i++)
    {
        int n;
        cin >> n;
        size_t sum = 1;
        size_t num;
        for (int j= 0; j < n; j++)
        {
            cin >> num;
            sum *= num;
        }

        sum += n - 1;
        cout << 2022 * sum << "\n";
    }
}
